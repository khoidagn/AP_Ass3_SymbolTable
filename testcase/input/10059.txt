INSERT x number
ASSIGN x 10.1