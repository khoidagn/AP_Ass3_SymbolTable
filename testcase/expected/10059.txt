Invalid: ASSIGN x 10.1